src/views/Workflow.vue

```diff
@@ -17,24 +17,47 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 <template>
   <div id="workflow-panel" class="fill-height">
-    <workflow
-      :workflow-name="workflowName"
-      :workflow-tree="tree"
-      :is-loading="isLoading"
-      ref="workflow-component" />
+    <lumino
+      ref="lumino"
+      v-on:lumino:deleted="onWidgetDeletedEvent"
+    >
+      <tree-component
+        v-for="widgetId of treeWidgets"
+        :key="widgetId"
+        :id="widgetId"
+        :workflows="tree.root.children"
+      />
+      <graph-component
+          v-for="widgetId of graphWidgets"
+          :key="widgetId"
+          :id="widgetId"
+          :workflow-name="workflowName"
+      />
+      <mutations-view
+          v-for="widgetId of mutationsWidgets"
+          :key="widgetId"
+          :id="widgetId"
+          :workflow-name="workflowName"
+      />
```

Comment:
Out of interest if the `:workflows`, `:workflow-name` interface were standardised between the components could we do this:

```vue
<component
  v-for="widget of widgets"
  :is="widget['type']"
  :key="widget['id']"
  :id="widget['id']"
  :workflow-names="[workflowName]"
  :tree="tree.root.children"
/>
```

Reply:
I think so! We could probably also have a unique `addWidget` method, instead of `addTreeWidget`, `addGraphWidget`, etc. So when the new `Dot` or `WorkflowTable` etc components are created, we don't need to remember to change this file (or if we do, it's a very small change in a single place I think)

Reply:
(Are you going to do this here, or is it for a follow-up PR?)

Reply:
Planning to leave for a follow-up PR if that's OK, and focus on just removing the multiple wrappers in this one.